decidim-verifications/app/views/decidim/verifications/authorizations/index.html.erb

```diff
@@ -3,7 +3,7 @@
 <% url_params = (redirect_url.present? ? { redirect_url: redirect_url } : {}) %>
 
 <% content_for :js_content do %>
-  <%= javascript_pack_tag "decidim_verifications" %>
+  <%= append_javascript_pack_tag "decidim_verifications" %>
```

Comment:
I don't think you need to wrap this inside `content_for`?

From other such tags it has been also removed.

Reply:
Yes, the content_for is not required.